src/Goteo/Console/EventListener/ConsoleWatcherListener.php

```diff
@@ -204,18 +204,19 @@ public function onProjectActive(FilterProjectEvent $event) {
             case 3: // Una acción a diario, por pequeña que sea
```

Comment:
I really think we should be using the constants that are in the Template class instead of using the hardcoded values of those constants.

Reply:
You are right, but since this values refer to days instead of values with other meanings... we could still create a DAY_X constant for every day.

Reply:
Alright, I really thought they were template IDs. And not days. Nevermind about this comment then.